modules/ROOT/pages/viewing-api-details.adoc

```diff
@@ -13,28 +13,45 @@ Name of the API.
 * *Type*
 + 
 The API type can be AsyncAPI, HTML, REST, or SOAP.
-* *Description*
-+
-A description of the API.
 * *Contact email*
 +
 The email of the contact for the API.
-* *Version*
+
+* *Categories*
++
+The name and value of the predefined category this API belongs to. APIs can have multiple categories. Categories are used to organize APIs into groups to improve browsing and discovery. Categories are created and maintained in Exchange.
+* *Tags*
++
+The name of a predefined keyword associated with this API. APIs can have multiple tags. Tags are used to filter search results. Tags are created and maintained in Exchange. Each minor version of the API has its own tags and categories. 
+* *Description*
++
+A description of the API.
+
+* *Asset version*
 +
 The version of the API.
-* *Instance*
+* *Version state*
++
+The lifecycle state of this API. Lifecycle states are development, stable, and deprecated.
+* *Instances*
 +
 The count of instances of the API.
-* *Client applications*
+* *Clients*
 +
 The number of client applications of the API.
 
-The API Console is located in the API Details page. The console component is interactive and shows information about API specifications and their methods. Endpoints are displayed in alphabetical order. See xref:api-console-specification-and-documentation.adoc[API Console Specification and Documentation] for more information.
+The API Console is located in the API Details page. The console component is interactive and shows information about API specifications and their methods. Endpoints are displayed in alphabetical order. See xref:using-the-api-console.adoc[Using the API Console] for more information.
+
+== View Categories and Tags
+To view the categories or tags for an API:
+
+. On the *API details* page, click *Categories* or *Tags* to view the information.
+. Click *Categories* or *Tags* again to close the list.
 
 == View Instances Information and Copy the URL
-To view the name and URL of an instance and copy the URL:
+To view the name and URL of an API instance and copy the URL:
 
-. Click *View instances*.
+. On the *API details* page, click *View instances*.
```

Comment:
```suggestion
. On the *API Details* page, click *View Instances*.
```

Reply:
Unless the UI has lowercase, in which case, suggest to the dev that it should be init cap...